### PR TITLE
Add Header component

### PR DESCRIPTION
**Pull request by ToobaJamal**
## Description

This PR adds a responsive header component with helpful structure and comments.

## Type of PR

- [x] ✨ Feature
- [ ] 🐞 Bug fix
- [ ] 📄 Documentation update
- [ ] Others

## Related Issue

#6 

### On desktop

header

### On tablet/ mobile

header1

### On tablet/ mobile when the bars icon is clicked

header2

### CODE REVIEW

File: src/components/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        </a>
+      </div>
+
+      {/* Mobile Nav Button */}
+
+      <div className='mobile-nav-button' onClick={() => setShowMobileNav(!showMobileNav)}>
+        <FaBars />
+      </div>
+
+      {/* Menu items list */}
```
Comment (ghost) [inline src/components/Header.jsx:29]: Thanks for including the helpful comments !

**Review by ghost — COMMENTED** [review]
Looks great @ToobaJamal ! I'll leave this open for a bit in case anyone else wants to review.

**Comment by Birdie13-stack** [comment]
I suggest adding a background color, and then pushing the main section a bit further down.

**Comment by ghost** [comment]
@Birdie13-stack Hi Birdie, thanks for the suggestion. Would you like to create an issue for this? Otherwise I will. Looking forward to seeing your footer component! :)

**Comment by Birdie13-stack** [comment]
Ok, I will. Nice work Maria

**Comment by ghost** [comment]
Thank you @Birdie13-stack !

**Pull request merged**